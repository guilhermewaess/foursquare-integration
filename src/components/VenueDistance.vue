<template>
  <v-layout column
            class="ma-0 primary"
            fill-height>
    <v-flex>
      <div class="text-xs-center white--text">Distance</div>
    </v-flex>
    <v-flex>
      <div class="white--text text-xs-center headline font-weight-medium">
        {{ location.distance }}m
      </div>
    </v-flex>
  </v-layout>
</template>

<script>
export default {
  name: 'VenueDistance',
  props: {
    location: {
      required: true,
      type: Object,
    },
  },
};
</script>
