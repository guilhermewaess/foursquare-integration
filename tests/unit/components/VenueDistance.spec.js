import { shallowMount } from '@vue/test-utils';
import VenueDistance from '@/components/VenueDistance.vue';

describe('VenueDistance', () => {
  let component;
  let location;
  beforeEach(() => {
    location = {
      distance: 100,
    };
    component = shallowMount(VenueDistance, {
      propsData: {
        location,
      },
      stubs: ['v-layout', 'v-flex'],
    });
  });
  it('should have name', () => {
    expect(component.name()).toEqual('VenueDistance');
  });
  describe('snapshot', () => {
    it('should render', () => {
      expect(component.element).toMatchSnapshot();
    });
  });
});
