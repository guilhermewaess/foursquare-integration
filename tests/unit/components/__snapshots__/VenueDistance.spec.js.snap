// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VenueDistance snapshot should render 1`] = `
<v-layout-stub
  class="ma-0 primary"
  column=""
  fill-height=""
>
  <v-flex-stub>
    <div
      class="text-xs-center white--text"
    >
      Distance
    </div>
  </v-flex-stub>
   
  <v-flex-stub>
    <div
      class="white--text text-xs-center headline font-weight-medium"
    >
      
      100m
    
    </div>
  </v-flex-stub>
</v-layout-stub>
`;
